lib/src/lints/avoid_returning_widgets/avoid_returning_widgets_rule.dart

```diff
@@ -96,7 +94,8 @@ class AvoidReturningWidgetsRule
 
       final isWidgetReturned = hasWidgetType(returnType);
 
-      final isIgnored = _shouldIgnore(node);
+      final isIgnored = config.parameters.matchMethod(node) ||
+          config.parameters.matchClass(node);
```

Comment:
The class_name parameter means that the method_name in the class_name should not be processed by this lint. Not the whole class.

Reply:
It does work this way.

If you only specify the class name, then the entire class will be ignored, but if you specify both class and method name, then only this method in this class will be ignored.

```yaml
exclude:
  # excludes a matching method in a matching class
  - method_name: excludeMethod
    class_name: ExcludeClass
  # excludes a matching method anywhere
  - method_name: excludeFunction
  # excludes all methods within a matching class
  - class_name: ExcludeEntireClass
```